Cubical/Data/Sigma/Base.agda

```diff
@@ -0,0 +1,8 @@
+{-# OPTIONS --cubical --safe #-}
+module Cubical.Data.Sigma.Base where
+
+open import Cubical.Core.Prelude
+
+-- TODO : Sigma-types are defined in Core/Prelude.
+-- should they be moved here ?
+-- Is it ok to have something in Core depending on something in Data ?
```

Comment:
We need Sigma for Glue types as far as I remember... It might be logical to re-export them here, so you can do a `public` import above. Maybe you should only import the Sigma type as well

Reply:
I think sigma types being defined in Core is fine. Results about sigma types should be in Data however. This is how the HoTT-Coq library works for example. Most types are defined in `Basics.Overture` but results about Types such as their paths are collected in `Types`.

Reply:
I agree with @Alizter . @loic-p Can you update the comment?